Update api-management-capacity.md (#118506)
Azure/azure-content: articles/api-management/api-management-capacity.md

+ API Management management plane services, such as management actions applied via the Azure portal or Azure Resource Manager, or load coming from the [developer portal](api-management-howto-developer-portal.md).
+ Selected operating system processes, including processes that involve cost of TLS handshakes on new connections.
+ Platform updates, such as OS updates on the underlying compute resources for the instance.

> + Number of APIs deployed, irregardless of activtity, which can consume additional capacity.

Reviewer: ```suggestion
+ Number of APIs deployed, regardless of activity, which can consume additional capacity.
```

> + Number of APIs deployed, irregardless of activtity, to an APIM instance can consume additional capacity.

Reviewer 2: ```suggestion
+ Number of APIs deployed, irregardless of activtity, which can consume additional capacity.
```
